infrastructure/crowdin/pushTranslations.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+#$3 github API Key
+pull_request() {
+  PR="{\"title\": \"[${BRANCH_NAME}] Translations update\",\"body\": \"Latest translations made in [Crowdin](https://crowdin.com/project/bonita-bpm)\n!skipTests\", \"head\": \"feat/${BRANCH_NAME}/update-translations\", \"base\": \"${BRANCH_NAME}\"}"
+  echo "Create new pull request $PR"
+  curl -i -X POST -d "$PR" \
+     https://api.github.com/repos/bonitasoft/bonita-web-pages/pulls?access_token=$1
+}
+
+
+scriptFolder=`pwd`/$(dirname $0)
+. $scriptFolder/utils.sh
```

Comment:
⚠️ the current script is not using functions declared in the `utils.sh` file
So I guess the `utils.sh` script should be removed